src/components/molecules/MoleculeSRPCard.vue

```diff
@@ -57,16 +65,23 @@ export default {
     filters: {
         available(slotAvailable) {
```

Comment:
do we need this function now?
I see the text is hard coded in the html component.